apps/api/src/app/events/services/chat-service/chat.factory.ts

```diff
@@ -23,3 +23,5 @@ export class ChatFactory implements IChatFactory {
     }
   }
 }
+
+// TODO remove?
```

Comment:
marked code with `// TODO remove?` which I'll revisit in the next PR and will check if this code can be removed